CustomerRegister/Repositories/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿
+
+using CustomerRegister.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CustomerRegister
+{
+    public class CustomerService : ICustomerService
+
+    {
+        private readonly List<CustomerEntity> _customers = new();
+
+        public bool Exists(CustomerEntity selectedCustomer)
+        {
+            var response = _customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email) && (selectedCustomer.Id != selectedCustomer.Id);
```

Comment:
```suggestion
            var response = _customers
                .Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email) && (selectedCustomer.Id != selectedCustomer.Id);
```
Para ficar visível todo o código, eu faria assim.